sbin/Makefile: Allow the sbin path to be configurable

Some project sharing ca-certificates from Debian allow configuration
of the installation location. Make the sbin location configurable.

Also ensure the target directory exists

Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>
Upstream-Status: Submitted [https://salsa.debian.org/debian/ca-certificates/-/merge_requests/5]

--- ca-certificates-20130119.orig/sbin/Makefile
+++ ca-certificates-20130119/sbin/Makefile
@@ -3,9 +3,12 @@
 #
 #

+SBINDIR = /usr/sbin
+
 all:

 clean:

 install:
-	install -m755 update-ca-certificates $(DESTDIR)/usr/sbin/
+	install -d $(DESTDIR)$(SBINDIR)
+	install -m755 update-ca-certificates $(DESTDIR)$(SBINDIR)/
